website/pages/docs/plugins/sources/aws/configuration.md

```diff
@@ -130,7 +130,7 @@ This is the (nested) spec used by the AWS source plugin.
   The region that should be used for signing the request to the endpoint
 
 
-## accounts
+## account
```

Comment:
Changing this from plural to singular because all of the org references are to the singular